src/widget/controlwidgetconnection.cpp

```diff
@@ -15,27 +15,26 @@ ControlWidgetConnection::ControlWidgetConnection(WBaseWidget* pBaseWidget,
     m_pControl->connectValueChanged(SLOT(slotControlValueChanged(double)));
 }
 
-ControlWidgetConnection::~ControlWidgetConnection() {
-}
+ControlWidgetConnection::~ControlWidgetConnection() = default;
```

Comment:
This is redundant in many ways. It overrides the already empty destructor of the base class, so it should be fine to delete this together with the prototype in the header file. 

In case of a destructor, the default keyword is always { }.  So there is IMHO no benefit of using it, it is longer, less explicit and requires C++11 knowledge to understand.  The { } notation has also the benefit that it provides a skeleton that can be filled if required. 

By the way: For the constructor case, the default keyword is very nice.